### PR TITLE
Release 1.1.0 with 1.17 and 1.18 support 

### PR DESCRIPTION
**Pull request by angelbarrera92**
This PR does:

- Update gatekeeper to beta9 (from beta8)
- Add compatibility against kubernetes 1.17 and 1.18 (http://ci.sighup.io/sighupio/fury-kubernetes-opa/75/3/1)
- Migrated from aws testing env to vsphere
- Release unreleased stuff (replicas 2 -> 1)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,8 +7,8 @@ Kubernetes Cluster.
 
 The following packages are included in Fury Kubernetes OPA module:
 
-- [Gatekeeper](katalog/gatekeeper): Ready to use gatekeeper deployment plus a set of rules. Version: **v3.1.0-beta.8**
-  - [Gatekeeper Core](katalog/gatekeeper/core): Gatekeeper deployment, ready to apply rules. Version: **v3.1.0-beta.8**
+- [Gatekeeper](katalog/gatekeeper): Ready to use gatekeeper deployment plus a set of rules. Version: **v3.1.0-beta.9**
```
Comment (phisco) [inline README.md:10]: Sure we want to support betas?

Comment (angelbarrera92) [reply README.md:10]: No, but we don't have any other alternative. Even on beta is the only version stable from OPA ;)

File: docs/releases/v1.1.0.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Release notes
+
+## Changelog
+
+Changes between `1.0.2` and this release: `1.1.0`
+
+- Add metrics port to service
+- Add service monitor definition for Prometheus scarping
```
Comment (ralgozino) [inline docs/releases/v1.1.0.md:8]: typo: scarping -> scraping

Comment (angelbarrera92) [reply docs/releases/v1.1.0.md]: Solved!

**Pull request merged**